x/supplier/keeper/msg_server_stake_supplier.go

```diff
@@ -130,7 +142,7 @@ func (k msgServer) StakeSupplier(ctx context.Context, msg *types.MsgStakeSupplie
 	}
 
 	// Send the coins from the message signer account to the staked supplier pool
-	err = k.bankKeeper.SendCoinsFromAccountToModule(ctx, msgSignerAddress, types.ModuleName, []sdk.Coin{coinsToEscrow})
+	err = k.bankKeeper.SendCoinsFromAccountToModule(ctx, msgSignerAddress, types.ModuleName, sdk.NewCoins(coinsToEscrow.Add(SupplierStakingFee)))
```

Comment:
Please use a separate small variable above that adds the two first w/ an appropriate name.